docs/src/main/sphinx/ext/download.py

```diff
@@ -20,8 +20,10 @@
 GROUP_ID = 'io.trino'
 ARTIFACTS = {
     'server': ('trino-server', 'tar.gz', None),
+    'server-rpm': ('trino-server-rpm', 'rpm', None),
```

Comment:
let's not make it easier to download RPM. it will move us further from our intended goal of removing it

Reply:
Okay .. but how are we supposed to tell readers where to get the RPM then? Thats all this does .. add a small link on the doc page. If we dont add that we have to describe that it is available in Maven Central .. or leave readers stranded and answer the question on slack again and again.

Reply:
Any input how to proceed @martint @findepi  ?